source/manual/fall-back-to-aws-cloudfront.html.md

```diff
@@ -9,29 +9,45 @@ parent: "/manual.html"
 
-**Important** At the time of writing, GOV.UK will be served
-by static mirrors and not the origin of GOV.UK if you fallback to the backup CDN.
-
-## Procedure
-
-You will have to make 2 DNS changes to GOV.UK:
-
-1. The CNAME at `www-cdn.production.govuk.service.gov.uk` should point to
-   cloudfront's WWW distribution instead of Fastly
-2. The CNAME at `assets.publishing.service.gov.uk` should point to cloudfront's
-   Assets distribution instead of Fastly
-
-This [Draft PR to Failover to AWS CloudFront](https://github.com/alphagov/govuk-dns-config/pull/673)
-shows the changes you need to make, and how to test the CDN before failing over.
-
-You can check the domain names of the cloudfront distributions by looking in
-the AWS web console, or with the CLI:
-
-```
-# www.gov.uk
-gds aws govuk-production-readonly aws cloudfront list-distributions --query "DistributionList.Items[?Comment=='WWW'].DomainName | [0]"
-
-# assets.publishing.service.gov.uk
-gds aws govuk-production-readonly aws cloudfront list-distributions --query "DistributionList.Items[?Comment=='Assets'].DomainName | [0]"
-```
-
+> **Important** GOV.UK will be served by static mirrors and not the origin of GOV.UK if you fallback to the backup CDN.
+> This is significantly degraded service compared to the primary CDN. Major features such as search, smart answers, and
+> postcode lookups will not work when using the secondary CDN.
+
+## Fail over checklist
+
+- Confirm that Fastly is the cause of the incident (check [https://status.fastly.com/](https://status.fastly.com/)
+  and keep an eye on twitter - if there's a major Fastly outage there will be a lot of noise)
+- Sign in to the AWS console as an admin (`gds aws govuk-production-admin -l`, or however you prefer to sign in to AWS)
+- Sign in to [the GCP console](https://console.cloud.google.com/home/dashboard?project=govuk-production)
+- Open the following four pages as separate tabs:
+  - [GCP Cloud DNS www-cdn.production.govuk.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/govuk-service-gov-uk/rrsets/www-cdn.production.govuk.service.gov.uk./CNAME/edit?project=govuk-production)
+  - [AWS Route 53 govuk.service.gov.uk](https://console.aws.amazon.com/route53/v2/hostedzones#ListRecordSets/Z22RPYZA77J620)
+  - [GCP Cloud DNS assets.publishing.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/publishing-service-gov-uk/rrsets/assets.publishing.service.gov.uk./CNAME/edit?project=govuk-production)
+  - [AWS Route 53 publishing.service.gov.uk](https://console.aws.amazon.com/route53/v2/hostedzones#ListRecordSets/Z3SBFBO09PD5HF)
+- Because the secondary CDN provides degraded service we have a grace period
+  before failing over
+  - For a major Fastly outage (more than 30% of requests failing) wait 15
+    minutes (from the start of the incident) before failing over
+  - For a minor Fastly outage (less than 30% of requests failing) wait at least
```

Comment:
This point is confusing to me, is it:
1. minor outage, wait 15 minutes, failover and contact SMT or
2. minor outage, wait 15 minutes, contact SMT -> they decide whether to failover?

Reply:
Yeah. I need to think about this more, it's too vague right now.

I think I was thinking "start a conversation with SMT immediately about how long to wait. Wait at least 15 minutes".

Reply:
I've clarified this point now (several months later 😬 )